_topic/android-restricted-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Restricted settings on Android."
+prompt: "Learn how to update restricted settings on Android for the Pebble app to work properly on Android 13+."
+keywords: "android app settings notifications"
+date:   2023-01-03 06:53:00
+author: "BlockArchitech"
+hideFirstStepNumber: false
+osSpecific: true
+os: android
+---
+
+# 1. Restricted Settings
+On Android 13 and higher, restricted settings prevents the Pebble app from working properly.
+
+
+![](https://raw.githubusercontent.com/BlockArchitech/temporary-images-repo/master/restricted.png)
+
+# 2. Open Settings
+
+Open the settings app on your device.
```

Comment:
I think this step could be merged with step 3.

"Open settings, find the apps menu and tap all apps"

It should also point out that on some Android distros things might look slightly different

Reply:
Will do!